7.js

```diff
@@ -0,0 +1,61 @@
+const months = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun',
+'Jul', 'Aug', 'Sep', 'Oct', 'Nov',
```

Comment:
The indentation here is a little incosistent.  Consider adding a two space indent to show that this is a continuation of the `months` declaration, and moving the next line (`'Dec' ]`) up to the previous line.